README.md

```diff
@@ -39,7 +39,7 @@ enabled = true
 slices = [ [13:10:00, 12:00:00], [19:00:10, 23:59:00] ] # you can write multiple time slice
 black_list = false # time slices are black list or white list?
 command = "mpv" # the command that application is running from
-args = "*vid*" # the argument of command(supports regex) and can be empty
+args = "*vid*" # the argument of command(supports *) and can be empty
```

Comment:
Shouldn't it be this?
```suggestion
args = ".*vid.*" # the argument of command(supports *) and can be empty
```

Reply:
I thought its not a good idea to get regex from user as it can make regexes that kill performance.

Reply:
Hmmm, I think it's okay, it's their system and they can do anything they want with it.